INSTALLATION.md

```diff
@@ -324,8 +324,8 @@ Follow these steps for setting up a software development environment.
          ```
       2. Running asynchronously in a subshell. You will have to use the `docker-compose down` command below to stop it.
          `bash
-      sudo /usr/libexec/docker/cli-plugins/docker-compose -f docker-compose.dev.yaml up --build &
-      `
+sudo /usr/libexec/docker/cli-plugins/docker-compose -f docker-compose.dev.yaml up --build &
```

Comment:
Your PR has too many unnecessary files that don’t relate to your issue. You can from the PR by running the commands below from the root directory of the repository

git add -u
git reset HEAD path/to/file
git commit

Please apply these changes to this file.